runtime/src/main/java/io/quarkiverse/quinoa/QuinoaDevWebSocketProxyHandler.java

```diff
@@ -68,7 +68,7 @@ public void handle(final RoutingContext ctx) {
 
                 httpClient.webSocket(options, clientContext -> {
                     if (clientContext.succeeded()) {
-                        LOG.debugf("Quinoa Dev WebSocket Client Connected: %s:%s%s", host, port, forwardUri);
+                        LOG.infof("Quinoa Dev WebSocket Client Connected: %s:%s%s", host, port, forwardUri);
```

Comment:
I made this an INFO because I like seeing when the HMR socket connects and plug it helps me in testing. Hope that is OK?